plugins/techdocs-backend/src/helpers.ts

```diff
@@ -146,10 +146,20 @@ export const checkoutGitRepository = async (
 
   if (token) {
     const type = getGitRepoType(repoUrl);
-    const auth = type === 'github' ? `${token}:x-oauth-basic` : `:${token}`;
-    parsedGitLocation.token = auth;
+    switch (type) {
+      case 'gitlab':
+        // Personal Access Token
+        parsedGitLocation.token = `dummyUsername:${token}`;
+        break;
+      case 'github':
+        parsedGitLocation.token = `${token}:x-oauth-basic`;
+        break;
+      default:
+        parsedGitLocation.token = `:${token}`;
+    }
   }
 
+  parsedGitLocation.git_suffix = true;
```

Comment:
This is the remaining question then - i'm kinda wondering if this will break github. If it was explicitly needed for gitlab, maybe you should move it into the gitlab specific path above?

Reply:
Updated